### ISSUE TITLE
train code

### ISSUE THREAD

**Issue by dangpnh2** [myblindcode/sentence_embeddings#1]
Hi,

Can you share the train code for this github?

**Comment by gubartz** [comment]
You can follow this tutorial

https://www.pinecone.io/learn/series/nlp/fine-tune-sentence-transformers-mnr/

But change the loss to [BatchAllTripletLoss](https://www.sbert.net/docs/package_reference/losses.html) and your dataset needs to be in (sentence, label) format instead of (anchor, positive).
